packages/bpk-docs/src/pages/NativeButtonPage/NativeButtonPage.js

```diff
@@ -137,6 +139,9 @@ const NativeTextPage = () => <DocsPageBuilder
     <Paragraph>
       Buttons can be text only, icon only, or text and icon.
     </Paragraph>,
+    <Paragraph>
+      Primary and secondary buttons can be <BpkLink href={THEMING}>themed</BpkLink>.
```

Comment:
This is duplicated in the readme as well as the page - I reckon we only need one.

Reply:
I've removed the one from the readme now